src/app/search/facet-search/facet-search-update.pipe.ts

```diff
@@ -12,17 +12,19 @@ import { Pipe, PipeTransform } from '@angular/core';
  * @memberof GetFacetSearchResultsPipe
  */
 export class GetFacetSearchUpdatePipe implements PipeTransform {
-  transform(unfilteredItems: Array<any>, searchText: string): string {
+  transform(unfilteredItems: Array<any>, searchText: string): string | null {
     let hiddenFilteredSize;
-    if (!searchText && unfilteredItems.length > 5) {
-      hiddenFilteredSize = unfilteredItems.length - 5;
+    const value = searchText.toLowerCase();
```

Comment:
Can searchText be null? This will fail if so.

Reply:
searchText can't be null, if empty it will be an empty string